### ISSUE TITLE
Why have a tensor of 1 or -1 in loss.backward()?

### ISSUE THREAD

**Issue by NeuronCrossroads** [martinarjovsky/WassersteinGAN#60]
What happens if we get rid of the mone and one in the .backward for the losses? What are they for?

**Comment by FlyingCarrot** [comment]
@NeuronCrossroads  The parameters passed into backward will control the radio of the final gradients, so the output.backward() and output.backward(1) does the same operations. However, the output.backward(-1) will change the gradients into an opposite direction.
